__docs__/wonder-blocks-core/view.stories.tsx

```diff
@@ -135,19 +136,19 @@ DefiningLayout.parameters = {
 
 const styles = StyleSheet.create({
     container: {
-        background: color.offBlack8,
+        background: semanticColor.core.background.neutral.subtle,
         gap: spacing.medium_16,
         padding: spacing.xLarge_32,
     },
 
     view: {
-        border: `1px dashed ${color.purple}`,
+        border: `1px dashed ${semanticColor.mastery.primary}`,
         gap: spacing.medium_16,
         padding: spacing.medium_16,
     },
 
     item: {
-        background: color.offBlack32,
+        background: semanticColor.core.background.disabled.strong,
```

Comment:
This story doesn't appear to have `disabled` characteristics...should it use a `disabled` token?

Reply:
Good point, I changed it to use `instructive.subtle`. Some extra stories might have changed.